dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
                                                  snr=None)
 
     return add_noise(S, snr, S0), sticks
+    
+def multi_tensor_dki(gtab, mevals, S0=100, angles=[(0., 0.), (90., 0.)],
+                         fractions=[50, 50], snr=20):
+    
+    r"""Simulate the elements of the diffusion and diffusion kurtosis tensor as
```

Comment:
Would be nice to have a one-line summary at the top, and then a more expansive explanation.